### PR TITLE
Mobile Map Visual Updates

### PR DESCRIPTION
**Pull request by Tvcz**
## Why

[ClickUp Ticket](https://app.clickup.com/t/27pkxm1)

Makes it easier to view and interact with maps, especially on mobile, allowing the use of one-finger map movement and reducing the obtrusiveness of overlayed map elements. The use of slide-downs allow for more of the map to be visible on mobile.

## This PR

- Changed map options for gestureHandling to "greedy" on applicable maps.

- Changed SlideDown to fit on screen properly and extended its use on mobile for a better user experience.

- Changed search bar size on mobile to allow for practical use.

- Changed tablet view to more closely align with mobile rather than desktop for maps. 

### Desktop landing page old vs new
<p float="left">
  
   
</p>

### Desktop my-trees page old vs new
<p float="left">
  
   
</p>

### Tablet landing page old vs new
<p float="left">
  
   
</p>

### Tablet my-trees page old vs new
<p float="left">
  
   
</p>

### Mobile landing page old vs new
<p float="left">
  
   
</p>

### Mobile my-trees page old vs new
<p float="left">
  
   
</p>

### Fixed Mobile map scrolling issue
<p float="left">
  
   
</p>

## Verification Steps

Visually inspected changes to landing map, my trees page, and site maps on desktop, tablet, and mobile sizes, and interacted with the maps to ensure they operated as intended.

### CODE REVIEW

**Review by rymaju — APPROVED** [review]
LGTM

File: src/components/mapComponents/maps/selectorMap/index.tsx [hunk 1]
```diff
@@ -63,6 +63,12 @@ const SelectorMap: React.FC<SelectorMapProps> = ({
       zoomListener: mapLayersAndListeners.zoomListener,
       markersArray: mapData.markersArray,
     };
+
+    mapData.map.setOptions({
+      gestureHandling: 'greedy',
+    });
```
Comment (SofieDunt) [inline src/components/mapComponents/maps/selectorMap/index.tsx:69]: 😍

Comment (SofieDunt) [reply src/components/mapComponents/maps/selectorMap/index.tsx:69]: ⛏️ Can you leave a comment explaining what this does?

File: src/components/navBar/index.tsx [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
+import {Col, Row} from 'antd';
+import useWindowDimensions, {WindowTypes} from '../windowDimensions';
 import MobileNavBar from './mobileNavBar';
-import { BACKGROUND_GREY, MID_GREEN } from '../../utils/colors';
+import {BACKGROUND_GREY, MID_GREEN} from '../../utils/colors';
 import sfttLogo from '../../assets/images/sfttNameLogo.png';
 import bostonLogo from '../../assets/images/bostonParksLogo.png';
 import c4cLogo from '../../assets/images/c4cTextLogo.png';
-import { LinkButton } from '../linkButton';
+import {LinkButton} from '../linkButton';
```
Comment (SofieDunt) [inline src/components/navBar/index.tsx:11]: ❓ Did prettier do this?

File: src/components/mapComponents/constants.tsx [hunk 3]
```diff
@@ -26,6 +26,9 @@ export const BOSTON_BOUNDS = {
 
 export const STREET_ZOOM = 19;
 
+export const MOBILE_SLIDE_HEIGHT = 60;
+export const DESKTOP_SLIDE_HEIGHT = 60;
```
Comment (SofieDunt) [inline src/components/mapComponents/constants.tsx:30]: ❓ Is there a reason to distinguish the two if they're both 60?

File: src/components/mapComponents/mapLegend/index.tsx [hunk 4]
```diff
@@ -69,7 +69,7 @@ interface MapLegendProps {
 const MapLegend: React.FC<MapLegendProps> = ({ view, mobile, canHide }) => {
   const [showLegend, setShowLegend] = useState(true);
 
-  const fontSize = `${mobile ? '10px' : '12px'}`;
+  const fontSize = `${mobile ? '15px' : '12px'}`;
```
Comment (SofieDunt) [inline src/components/mapComponents/mapLegend/index.tsx:72]: 💭 Maybe we should also make these font sizes a constant

**Review by SofieDunt — APPROVED** [review]
💯 Looks great! Nice job with the map

File: src/components/mapComponents/constants.tsx (hunk 3 above)
Comment (Tvcz) [reply src/components/mapComponents/constants.tsx:30]: whoops, desktop should be 70. I'll fix that.

**Pull request merged**